src/highrl/envs/robot_env.py

```diff
@@ -101,6 +102,8 @@ def __init__(
                     "wall_time",
                 ]
             )
+        self.robot_episode_reward_writer = SummaryWriter(log_dir="runs")
```

Comment:
Create global variable for folder path instead for hard coding the value, for future easy manipulation.

Reply:
Done